frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-page.tsx

```diff
@@ -225,12 +252,17 @@ export const OperatorHubPage = withFallback(
                     { key: 'opsrc-owner-name', operator: 'DoesNotExist' },
                     { key: 'csc-owner-name', operator: 'DoesNotExist' },
                   ]),
-                  prop: 'packageManifest',
+                  prop: 'packageManifests',
                 },
                 {
                   isList: true,
                   kind: referenceForModel(SubscriptionModel),
-                  prop: 'subscription',
+                  prop: 'subscriptions',
+                },
+                {
+                  isList: true,
+                  kind: referenceForModel(ClusterServiceVersionModel),
+                  prop: 'clusterServiceVersions',
                 },
```

Comment:
This is a really expensive watch. CSVs contain a lot of data, and then they're copied across all namespaces. Are we able to wait until the user clicks the tile and get the single CSV we need to show the version?

Reply:
Yeah, we could feasibly do that.